### PR TITLE
Public Profile - Farcaster Follower Count

### PR DESCRIPTION
**Pull request by JacobHomanics**
Client to fetch farcaster user data.

Hook that utilizes the client to retrieve the data.

Satisfies https://github.com/voteagora/op-atlas/issues/427

### CODE REVIEW

File: app/src/components/profile/public/ProfileHeaderLinks.tsx [hunk 1]
```diff
@@ -2,10 +2,19 @@ import BubbleLink from "@/components/common/BubbleLink"
 import useDelegateData from "@/hooks/api/useDelegateData"
 import { UserWithAddresses } from "@/lib/types"
 
+function formatNumber(num: number): string {
+  if (num >= 1000) {
+    return (num / 1000).toFixed(1).replace(/\.0$/, "") + "k"
+  }
+  return num.toString()
+}
+
```
Comment (stepandel) [inline app/src/components/profile/public/ProfileHeaderLinks.tsx:11]: same comments as with github data -- let' use `formatNumber` from the `utils`

File: app/src/components/profile/public/ProfileHeaderLinks.tsx [hunk 2]
```diff
@@ -19,7 +28,14 @@ export default function ProfileHeaderLinks({
       <BubbleLink
         href={`https://warpcast.com/${user.username}`}
         icon="/assets/icons/farcaster-icon.svg"
-        text={<span className="text-sm text-black">@{user.username}</span>}
+        text={
+          <>
+            <span className="text-sm text-black">@{user.username}</span>
+            <span className="text-sm text-[#404454]">
```
Comment (stepandel) [inline app/src/components/profile/public/ProfileHeaderLinks.tsx:34]:
```suggestion
            <span className="text-sm text-gray-500">
```

File: app/src/lib/actions/neynar.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      return {
+        error: null,
+        user: user ?? null,
+      }
+    }
+  } catch (error: unknown) {
+    console.error("Error searching for user", (error as Error).message)
+    return {
+      error: "Error searching for user",
+    }
```
Comment (stepandel) [inline app/src/lib/actions/neynar.ts:16]: this return error pattern works well when we want to inform user of an error. Fetching github data does not require user input and can / should fail silently. It's okay to send these error directly into dev console. External API calls should be able to fail without degrading user experience
tldr: let's remove the try / catch block

**Review by stepandel — CHANGES_REQUESTED** [review]
small nits

**Comment by JacobHomanics** [comment]
Awaiting response on https://github.com/voteagora/op-atlas/pull/429#issuecomment-2568366982 as the code in this PR follows the same principles.

**Comment by JacobHomanics** [comment]
should be gtg!

**Review by stepandel — APPROVED** [review]
